src/DependencyInjection/SonataUserExtension.php

```diff
@@ -69,12 +69,23 @@ public function load(array $configs, ContainerBuilder $container): void
 
         $this->checkManagerTypeToModelTypesMapping($config);
 
+        $resetting = false;
+
         $this->configureClass($config, $container);
-        $this->configureMailer($config, $container);
         $this->configureDefaultAvatar($config['profile'], $container);
         if (isset($bundles['SonataAdminBundle'])) {
             $this->configureAdmin($config['admin'], $container);
-            $this->configureResetting($config['resetting'], $container);
+            // check for reset configuration
+            if (isset($config['resetting']['email']['address'], $config['resetting']['email']['sender_name'])) {
```

Comment:
IMO this is missing a check to really validate the user has required deps (since they will be optional on require-dev) wdyt ?